kotskinds/schemas/helmchart-kots-v1beta1.json

```diff
@@ -22,7 +22,37 @@
       "properties": {
         "builder": {
           "type": "object",
-          "additionalProperties": {}
+          "additionalProperties": {
```

Comment:
proves my point above ^

Reply:
this file needs to be regenerated

Reply:
I'm not sure why this is still here, I've regenerated these files and the only place these additional properties appears is in the `MappedChartValue` struct and the `// +kubebuilder:validation:Type=""` notation doesn't seem to affect the generation.